packages/web_ui/src/components/AssignInstanceModal.tsx

```diff
@@ -77,13 +77,14 @@ export default function AssignInstanceModal(props: AssignInstanceModalProps) {
 			</Paragraph>
 			<Form form={form} initialValues={{ host: props.hostId ?? "null" }}>
 				<Form.Item name="host" label="Host">
-					<Select>
+					<Select showSearch optionFilterProp="name">
 						<Select.Option value={"null"}>
```

Comment:
Should be mentioned that the <Select.Option> syntax is deprecated in antd - the new API is just an array to <Select /> instead. Not a big deal to update whenever they release v6 though.

Reply:
Noted, I also expect other things will break when updating to v6 so we can do all changes in a single PR.